test/e2e/suites/managed/eks_test.go

```diff
@@ -136,7 +137,13 @@ var _ = ginkgo.Describe("[managed] [general] EKS cluster tests", func() {
 			Name:      clusterName,
 		})
 		Expect(cluster).NotTo(BeNil(), "couldn't find CAPI cluster")
-
+		clusterClient := e2eCtx.Environment.BootstrapClusterProxy.GetWorkloadCluster(ctx, namespace.Name, clusterName).GetClient()
+		ginkgo.By("Creating the LB service")
```

Comment:
It may not be necessary for the test but it just feels strange to me to create a service without the target deployment or a pod when the service is targeting pods with `"app": "nginx"` selector.

Reply:
Let me see if the `unamanaged functional` tests are doing this as well. If not I'll add them in both places.